pkg/storage/utils/decomposedfs/decomposedfs.go

```diff
@@ -274,16 +274,21 @@ func (fs *Decomposedfs) Postprocessing(ch <-chan events.Event) {
 				continue // NOTE: since we can't get the upload, we can't delete the blob
 			}
 
-			var (
-				failed     bool
-				keepUpload bool
-			)
-
 			n, err := session.Node(ctx)
 			if err != nil {
 				log.Error().Err(err).Str("uploadID", ev.UploadID).Msg("could not read node")
 				continue
 			}
+			if !n.Exists {
+				log.Debug().Str("uploadID", ev.UploadID).Str("nodeID", session.NodeID()).Msg("node no longer exists")
```

Comment:
Here we have to handle the case when a file has been deleted before the postprocessing.
Besides the cleanup, we should publish a proper event. 
@butonic @kobergj Please suggest what it could be.

Reply:
Why do we need to emit an event? Who will react to this event?